.eslintrc

```diff
@@ -7,6 +7,7 @@
   ],
   "env": {
     "browser": true,
+    "es6": true,
```

Comment:
this may not be needed, my local development linter was complaining after i did npm install, which this fixed. I will refresh my editor without this rule and see if it's still needed.